solutions/10.js

```diff
@@ -0,0 +1,34 @@
+// Tri Le
+// Find sum of prime numbers 
+
+const isPrime = (number) => {
+  if(number === 2){
+    return true;
+  }
+  else if (number < 2){
+    return false;
+  }
+  else {
```

Comment:
Don't need else statement here. Return stops the function, so if the code is still running, none of the cases matched.